src/exception.cc

```diff
@@ -1,5 +1,6 @@
 /* 
  * Copyright 2010-2011 PathScale, Inc. All rights reserved.
+ * Copyright 2021 David Chisnall, Inc. All rights reserved.
```

Comment:
, Inc? 🙄

Reply:
```suggestion
 * Copyright 2021 David Chisnall. All rights reserved.
```

Ooops, copy and paste fail.